### ISSUE TITLE
Fix deprecated usages of `include`

### ISSUE THREAD

**Issue by cpu** [StreisandEffect/streisand#959]
Using Ansible 2.4.0.0 there are some new deprecation warnings Streisand should address:

```
 [WARNING]: Found both group and host with same name: localhost

[DEPRECATION WARNING]: 'include' for playbook includes. You should use 'import_playbook' instead. This feature will be removed in version 2.8. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: The use of 'include' for tasks has been deprecated. Use 'import_tasks' for static inclusions or 'include_tasks' for dynamic inclusions. This feature will be removed in a future release. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: include is kept for backwards compatibility but usage is discouraged. The module documentation details page may explain more about this rationale.. This feature will be removed in a future release. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
```

**Comment by cpu** [comment]
It might be fairly annoying to fix this. `include_tasks`, `import_tasks`, `import_roles` and `import_plays` are all new in the same version that deprecates `include` :roll_eyes:  We would need to move our ansible required version > 2.4.0.

**Comment by wzyboy** [comment]
The deprecation warnings of `include` syntax should go away after https://github.com/StreisandEffect/streisand/pull/996 is merged.

**Comment by cpu** [comment]
Resolved with #996
